h2o-genmodel/src/test/resources/hex/genmodel/algos/ensemble/regression/models/GBM/GBM_model_R_1510186395165_2947/experimental/README.md

```diff
@@ -0,0 +1,2 @@
+Outputting model information in JSON is an experimental feature and we appreciate any feedback.
```

Comment:
I don't think we need the "experimental" directories in Git.

Reply:
Correct, I can remove.